cl/people_db/models.py

```diff
@@ -1169,6 +1169,14 @@ class Source(models.Model):
         blank=True,
     )
 
+    def save(self, *args, **kwargs):
```

Comment:
Neat, but why can't aliases have source info? We might wonder where we learned that Michael went by Mike, right?

Reply:
Yep.  I'll update this as well.  Thanks!

Reply:
Do we need this function here for some reason if we remove the other validation rule that you had? Seems to me (correct me if I'm wrong), that this save method can be nixed too?